plugin/trino-hive/pom.xml

```diff
@@ -545,6 +545,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
I'm thinking this will not be a welcome addition. The test _should_ all be using junit 5, aka "jupiter" now, no?

Reply:
Thanks for catching this. This was recommended by the dependency-analyze task in the build process. I rerun the build without this dependency, and the build was succeed. This should be removed.